dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/SchedulerServiceImpl.java

```diff
@@ -171,6 +171,9 @@ public Map<String, Object> insertSchedule(User loginUser,
         scheduleObj.setProcessDefinitionName(processDefinition.getName());
 
         ScheduleParam scheduleParam = JSONUtils.parseObject(schedule, ScheduleParam.class);
+        if (now.after(scheduleParam.getStartTime())) {
+            scheduleParam.setStartTime(now);
+        }
```

Comment:
could we refactor to a private method?

Reply:
Hi, @DarkAssassinator, i have thought again, maybe there's not a strong need to move this simple condition judgement into a separate method, like other `startTime` related checks, WDYT?